test/userrepo-test.js

```diff
@@ -1,16 +1,20 @@
 import { expect } from 'chai';
-
 import UserRepo from '../src/User-repo';
 import User from '../src/User';
 
-
-describe('User Repo', function() {
+describe('User Repo', () => {
   let user1;
```

Comment:
I think you can write this on one line: let user1, user2, user3, ....... user4000;